inappmessaging/USERGUIDE.md

```diff
@@ -41,20 +42,22 @@ Launches the URI in an external  browser on tap.
 Launches the URI in an external  browser on tap.
 
 ### Triggers
-This are events sent from the host app which are used as conditions if a specific campaign/message should be displayed.
+These are events sent from the host app that are used as conditions if a specific campaign/message should be displayed.
```

Comment:
used as conditions if -> used as conditions to determine if

Reply:
Updated